### PR TITLE
Section 3 text improvement

### PR DESCRIPTION
**Pull request by sunghee-hwang**
***

### :boom: Error: 400 Bad Request :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Jul 17, 2023, 2:20 AM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [CSS Spec Preprocessor](https://api.csswg.org/bikeshed/) - CSS Spec Preprocessor is the web service used to build Bikeshed specs.

:link: [Related URL](https://api.csswg.org/bikeshed/?url=https%3A%2F%2Fraw.githubusercontent.com%2FAOMediaCodec%2Fiamf%2F020792601dbc8e719ea286eb2a38c51463530447%2Findex.bs&md-warning=not%20ready)

```
Error running preprocessor, returned code: 2.
Your document appears to use tabs to indent, but line 394 starts with spaces.
Your document appears to use tabs to indent, but line 396 starts with spaces.
Your document appears to use tabs to indent, but line 397 starts with spaces.
Your document appears to use tabs to indent, but line 398 starts with spaces.
Your document appears to use tabs to indent, but line 399 starts with spaces.
Your document appears to use tabs to indent, but line 400 starts with spaces.
Your document appears to use tabs to indent, but line 401 starts with spaces.
Your document appears to use tabs to indent, but line 402 starts with spaces.
Your document appears to use tabs to indent, but line 403 starts with spaces.
Your document appears to use tabs to indent, but line 404 starts with spaces.
Your document appears to use tabs to indent, but line 405 starts with spaces.
Your document appears to use tabs to indent, but line 406 starts with spaces.
Your document appears to use tabs to indent, but line 407 starts with spaces.
Your document appears to use tabs to indent, but line 408 starts with spaces.
Your document appears to use tabs to indent, but line 409 starts with spaces.
Your document appears to use tabs to indent, but line 410 starts with spaces.
Your document appears to use tabs to indent, but line 411 starts with spaces.
Your document appears to use tabs to indent, but line 412 starts with spaces.
Your document appears to use tabs to indent, but line 413 starts with spaces.
Your document appears to use tabs to indent, but line 428 starts with spaces.
Your document appears to use tabs to indent, but line 429 starts with spaces.
Your document appears to use tabs to indent, but line 430 starts with spaces.
Your document appears to use tabs to indent, but line 431 starts with spaces.
Your document appears to use tabs to indent, but line 432 starts with spaces.
Your document appears to use tabs to indent, but line 434 starts with spaces.
Your document appears to use tabs to indent, but line 435 starts with spaces.
Your document appears to use tabs to indent, but line 436 starts with spaces.
Your document appears to use tabs to indent, but line 437 starts with spaces.
Your document appears to use tabs to indent, but line 438 starts with spaces.
Your document appears to use tabs to indent, but line 439 starts with spaces.
Your document appears to use tabs to indent, but line 440 starts with spaces.
Your document appears to use tabs to indent, but line 441 starts with spaces.
Your document appears to use tabs to indent, but line 451 starts with spaces.
Your document appears to use tabs to indent, but line 452 starts with spaces.
Your document appears to use tabs to indent, but line 453 starts with spaces.
Your document appears to use tabs to indent, but line 454 starts with spaces.
Your document appears to use tabs to indent, but line 455 starts with spaces.
Your document appears to use tabs to indent, but line 456 starts with spaces.
Your document appears to use tabs to indent, but line 457 starts with spaces.
Your document appears to use tabs to indent, but line 458 starts with spaces.
Your document appears to use tabs to indent, but line 459 starts with spaces.
Your document appears to use tabs to indent, but line 522 starts with spaces.
Your document appears to use tabs to indent, but line 523 starts with spaces.
Your document appears to use tabs to indent, but line 524 starts with spaces.
Your document appears to use tabs to indent, but line 553 starts with spaces.
Your document appears to use tabs to indent, but line 554 starts with spaces.
Your document appears to use tabs to indent, but line 558 starts with spaces.
Your document appears to use tabs to indent, but line 559 starts with spaces.
Your document appears to use tabs to indent, but line 560 starts with spaces.
Your document appears to use tabs to indent, but line 561 starts with spaces.
Your document appears to use tabs to indent, but line 597 starts with spaces.
Your document appears to use tabs to indent, but line 598 starts with spaces.
Your document appears to use tabs to indent, but line 599 starts with spaces.
Your document appears to use tabs to indent, but line 600 starts with spaces.
Your document appears to use tabs to indent, but line 601 starts with spaces.
Your document appears to use tabs to indent, but line 603 starts with spaces.
Your document appears to use tabs to indent, but line 604 starts with spaces.
Your document appears to use tabs to indent, but line 605 starts with spaces.
Your document appears to use tabs to indent, but line 606 starts with spaces.
Your document appears to use tabs to indent, but line 607 starts with spaces.
Your document appears to use tabs to indent, but line 608 starts with spaces.
Your document appears to use tabs to indent, but line 609 starts with spaces.
Your document appears to use tabs to indent, but line 610 starts with spaces.
Your document appears to use tabs to indent, but line 611 starts with spaces.
Your document appears to use tabs to indent, but line 612 starts with spaces.
Your document appears to use tabs to indent, but line 613 starts with spaces.
Your document appears to use tabs to indent, but line 614 starts with spaces.
Your document appears to use tabs to indent, but line 615 starts with spaces.
Your document appears to use tabs to indent, but line 616 starts with spaces.
Your document appears to use tabs to indent, but line 617 starts with spaces.
Your document appears to use tabs to indent, but line 618 starts with spaces.
Your document appears to use tabs to indent, but line 619 starts with spaces.
Your document appears to use tabs to indent, but line 620 starts with spaces.
Your document appears to use tabs to indent, but line 621 starts with spaces.
Your document appears to use tabs to indent, but line 623 starts with spaces.
Your document appears to use tabs to indent, but line 624 starts with spaces.
Your document appears to use tabs to indent, but line 625 starts with spaces.
Your document appears to use tabs to indent, but line 626 starts with spaces.
Your document appears to use tabs to indent, but line 627 starts with spaces.
Your document appears to use tabs to indent, but line 628 starts with spaces.
Your document appears to use tabs to indent, but line 629 starts with spaces.
Your document appears to use tabs to indent, but line 630 starts with spaces.
Your document appears to use tabs to indent, but line 636 starts with spaces.
Your document appears to use tabs to indent, but line 642 starts with spaces.
Your document appears to use tabs to indent, but line 643 starts with spaces.
Your document appears to use tabs to indent, but line 661 starts with spaces.
Your document appears to use tabs to indent, but line 662 starts with spaces.
Your document appears to use tabs to indent, but line 663 starts with spaces.
Your document appears to use tabs to indent, but line 703 starts with spaces.
Your document appears to use tabs to indent, but line 706 starts with spaces.
Your document appears to use tabs to indent, but line 709 starts with spaces.
Your document appears to use tabs to indent, but line 712 starts with spaces.
Your document appears to use tabs to indent, but line 749 starts with spaces.
Your document appears to use tabs to indent, but line 750 starts with spaces.
Your document appears to use tabs to indent, but line 751 starts with spaces.
Your document appears to use tabs to indent, but line 752 starts with spaces.
Your document appears to use tabs to indent, but line 753 starts with spaces.
Your document appears to use tabs to indent, but line 754 starts with spaces.
Your document appears to use tabs to indent, but line 755 starts with spaces.
Your document appears to use tabs to indent, but line 756 starts with spaces.
Your document appears to use tabs to indent, but line 757 starts with spaces.
Your document appears to use tabs to indent, but line 758 starts with spaces.
Your document appears to use tabs to indent, but line 759 starts with spaces.
Your document appears to use tabs to indent, but line 760 starts with spaces.
Your document appears to use tabs to indent, but line 761 starts with spaces.
Your document appears to use tabs to indent, but line 776 starts with spaces.
Your document appears to use tabs to indent, but line 777 starts with spaces.
Your document appears to use tabs to indent, but line 778 starts with spaces.
Your document appears to use tabs to indent, but line 779 starts with spaces.
Your document appears to use tabs to indent, but line 780 starts with spaces.
Your document appears to use tabs to indent, but line 781 starts with spaces.
Your document appears to use tabs to indent, but line 782 starts with spaces.
Your document appears to use tabs to indent, but line 783 starts with spaces.
Your document appears to use tabs to indent, but line 784 starts with spaces.
Your document appears to use tabs to indent, but line 785 starts with spaces.
Your document appears to use tabs to indent, but line 786 starts with spaces.
Your document appears to use tabs to indent, but line 787 starts with spaces.
Your document appears to use tabs to indent, but line 788 starts with spaces.
Your document appears to use tabs to indent, but line 789 starts with spaces.
Your document appears to use tabs to indent, but line 790 starts with spaces.
Your document appears to use tabs to indent, but line 828 starts with spaces.
Your document appears to use tabs to indent, but line 829 starts with spaces.
Your document appears to use tabs to indent, but line 830 starts with spaces.
Your document appears to use tabs to indent, but line 831 starts with spaces.
Your document appears to use tabs to indent, but line 832 starts with spaces.
Your document appears to use tabs to indent, but line 836 starts with spaces.
Your document appears to use tabs to indent, but line 837 starts with spaces.
Your document appears to use tabs to indent, but line 838 starts with spaces.
Your document appears to use tabs to indent, but line 839 starts with spaces.
Your document appears to use tabs to indent, but line 840 starts with spaces.
Your document appears to use tabs to indent, but line 841 starts with spaces.
Your document appears to use tabs to indent, but line 842 starts with spaces.
Your document appears to use tabs to indent, but line 843 starts with spaces.
Your document appears to use tabs to indent, but line 844 starts with spaces.
Your document appears to use tabs to indent, but line 845 starts with spaces.
Your document appears to use tabs to indent, but line 846 starts with spaces.
Your document appears to use tabs to indent, but line 885 starts with spaces.
Your document appears to use tabs to indent, but line 886 starts with spaces.
Your document appears to use tabs to indent, but line 887 starts with spaces.
Your document appears to use tabs to indent, but line 888 starts with spaces.
Your document appears to use tabs to indent, but line 889 starts with spaces.
Your document appears to use tabs to indent, but line 890 starts with spaces.
Your document appears to use tabs to indent, but line 891 starts with spaces.
Your document appears to use tabs to indent, but line 892 starts with spaces.
Your document appears to use tabs to indent, but line 893 starts with spaces.
Your document appears to use tabs to indent, but line 894 starts with spaces.
Your document appears to use tabs to indent, but line 895 starts with spaces.
Your document appears to use tabs to indent, but line 942 starts with spaces.
Your document appears to use tabs to indent, but line 943 starts with spaces.
Your document appears to use tabs to indent, but line 944 starts with spaces.
Your document appears to use tabs to indent, but line 945 starts with spaces.
Your document appears to use tabs to indent, but line 946 starts with spaces.
Your document appears to use tabs to indent, but line 947 starts with spaces.
Your document appears to use tabs to indent, but line 962 starts with spaces.
Your document appears to use tabs to indent, but line 963 starts with spaces.
Your document appears to use tabs to indent, but line 964 starts with spaces.
Your document appears to use tabs to indent, but line 965 starts with spaces.
Your document appears to use tabs to indent, but line 966 starts with spaces.
Your document appears to use tabs to indent, but line 967 starts with spaces.
Your document appears to use tabs to indent, but line 971 starts with spaces.
Your document appears to use tabs to indent, but line 972 starts with spaces.
Your document appears to use tabs to indent, but line 973 starts with spaces.
Your document appears to use tabs to indent, but line 977 starts with spaces.
Your document appears to use tabs to indent, but line 978 starts with spaces.
Your document appears to use tabs to indent, but line 979 starts with spaces.
Your document appears to use tabs to indent, but line 980 starts with spaces.
Your document appears to use tabs to indent, but line 990 starts with spaces.
Your document appears to use tabs to indent, but line 991 starts with spaces.
Your document appears to use tabs to indent, but line 1026 starts with spaces.
Your document appears to use tabs to indent, but line 1027 starts with spaces.
Your document appears to use tabs to indent, but line 1028 starts with spaces.
Your document appears to use tabs to indent, but line 1029 starts with spaces.
Your document appears to use tabs to indent, but line 1030 starts with spaces.
Your document appears to use tabs to indent, but line 1031 starts with spaces.
Your document appears to use tabs to indent, but line 1032 starts with spaces.
Your document appears to use tabs to indent, but line 1033 starts with spaces.
Your document appears to use tabs to indent, but line 1035 starts with spaces.
Your document appears to use tabs to indent, but line 1036 starts with spaces.
Your document appears to use tabs to indent, but line 1037 starts with spaces.
Your document appears to use tabs to indent, but line 1038 starts with spaces.
Your document appears to use tabs to indent, but line 1039 starts with spaces.
Your document appears to use tabs to indent, but line 1040 starts with spaces.
Your document appears to use tabs to indent, but line 1041 starts with spaces.
Your document appears to use tabs to indent, but line 1042 starts with spaces.
Your document appears to use tabs to indent, but line 1043 starts with spaces.
Your document appears to use tabs to indent, but line 1044 starts with spaces.
Your document appears to use tabs to indent, but line 1045 starts with spaces.
Your document appears to use tabs to indent, but line 1046 starts with spaces.
Your document appears to use tabs to indent, but line 1047 starts with spaces.
Your document appears to use tabs to indent, but line 1048 starts with spaces.
Your document appears to use tabs to indent, but line 1049 starts with spaces.
Your document appears to use tabs to indent, but line 1050 starts with spaces.
Your document appears to use tabs to indent, but line 1051 starts with spaces.
Your document appears to use tabs to indent, but line 1052 starts with spaces.
Your document appears to use tabs to indent, but line 1053 starts with spaces.
Your document appears to use tabs to indent, but line 1096 starts with spaces.
Your document appears to use tabs to indent, but line 1105 starts with spaces.
Your document appears to use tabs to indent, but line 1119 starts with spaces.
Your document appears to use tabs to indent, but line 1135 starts with spaces.
Your document appears to use tabs to indent, but line 1136 starts with spaces.
Your document appears to use tabs to indent, but line 1137 starts with spaces.
Your document appears to use tabs to indent, but line 1138 starts with spaces.
Your document appears to use tabs to indent, but line 1167 starts with spaces.
Your document appears to use tabs to indent, but line 1173 starts with spaces.
Your document appears to use tabs to indent, but line 1192 starts with spaces.
Your document appears to use tabs to indent, but line 1209 starts with spaces.
Your document appears to use tabs to indent, but line 1210 starts with spaces.
Your document appears to use tabs to indent, but line 1211 starts with spaces.
Your document appears to use tabs to indent, but line 1212 starts with spaces.
Your document appears to use tabs to indent, but line 1213 starts with spaces.
Your document appears to use tabs to indent, but line 1214 starts with spaces.
Your document appears to use tabs to indent, but line 1215 starts with spaces.
Your document appears to use tabs to indent, but line 1216 starts with spaces.
Your document appears to use tabs to indent, but line 1217 starts with spaces.
Your document appears to use tabs to indent, but line 1227 starts with spaces.
Your document appears to use tabs to indent, but line 1228 starts with spaces.
Your document appears to use tabs to indent, but line 1229 starts with spaces.
Your document appears to use tabs to indent, but line 1238 starts with spaces.
Your document appears to use tabs to indent, but line 1239 starts with spaces.
Your document appears to use tabs to indent, but line 1240 starts with spaces.
Your document appears to use tabs to indent, but line 1241 starts with spaces.
Your document appears to use tabs to indent, but line 1242 starts with spaces.
Your document appears to use tabs to indent, but line 1243 starts with spaces.
Your document appears to use tabs to indent, but line 1244 starts with spaces.
Your document appears to use tabs to indent, but line 1245 starts with spaces.
Your document appears to use tabs to indent, but line 1246 starts with spaces.
Your document appears to use tabs to indent, but line 1247 starts with spaces.
Your document appears to use tabs to indent, but line 1248 starts with spaces.
Your document appears to use tabs to indent, but line 1249 starts with spaces.
Your document appears to use tabs to indent, but line 1250 starts with spaces.
Your document appears to use tabs to indent, but line 1251 starts with spaces.
Your document appears to use tabs to indent, but line 1265 starts with spaces.
Your document appears to use tabs to indent, but line 1266 starts with spaces.
Your document appears to use tabs to indent, but line 1267 starts with spaces.
Your document appears to use tabs to indent, but line 1269 starts with spaces.
Your document appears to use tabs to indent, but line 1270 starts with spaces.
Your document appears to use tabs to indent, but line 1271 starts with spaces.
Your document appears to use tabs to indent, but line 1273 starts with spaces.
Your document appears to use tabs to indent, but line 1274 starts with spaces.
Your document appears to use tabs to indent, but line 1275 starts with spaces.
Your document appears to use tabs to indent, but line 1276 starts with spaces.
Your document appears to use tabs to indent, but line 1277 starts with spaces.
Your document appears to use tabs to indent, but line 1278 starts with spaces.
Your document appears to use tabs to indent, but line 1279 starts with spaces.
Your document appears to use tabs to indent, but line 1280 starts with spaces.
Your document appears to use tabs to indent, but line 1281 starts with spaces.
Your document appears to use tabs to indent, but line 1282 starts with spaces.
Your document appears to use tabs to indent, but line 1283 starts with spaces.
Your document appears to use tabs to indent, but line 1284 starts with spaces.
Your document appears to use tabs to indent, but line 1293 starts with spaces.
Your document appears to use tabs to indent, but line 1294 starts with spaces.
Your document appears to use tabs to indent, but line 1295 starts with spaces.
Your document appears to use tabs to indent, but line 1296 starts with spaces.
Your document appears to use tabs to indent, but line 1310 starts with spaces.
Your document appears to use tabs to indent, but line 1311 starts with spaces.
Your document appears to use tabs to indent, but line 1312 starts with spaces.
Your document appears to use tabs to indent, but line 1313 starts with spaces.
Your document appears to use tabs to indent, but line 1336 starts with spaces.
Your document appears to use tabs to indent, but line 1337 starts with spaces.
Your document appears to use tabs to indent, but line 1338 starts with spaces.
Your document appears to use tabs to indent, but line 1339 starts with spaces.
Your document appears to use tabs to indent, but line 1340 starts with spaces.
Your document appears to use tabs to indent, but line 1341 starts with spaces.
Your document appears to use tabs to indent, but line 1342 starts with spaces.
Your document appears to use tabs to indent, but line 1343 starts with spaces.
Your document appears to use tabs to indent, but line 1344 starts with spaces.
Your document appears to use tabs to indent, but line 1345 starts with spaces.
Your document appears to use tabs to indent, but line 1346 starts with spaces.
Your document appears to use tabs to indent, but line 1347 starts with spaces.
Your document appears to use tabs to indent, but line 1348 starts with spaces.
Your document appears to use tabs to indent, but line 1350 starts with spaces.
Your document appears to use tabs to indent, but line 1351 starts with spaces.
Your document appears to use tabs to indent, but line 1352 starts with spaces.
Your document appears to use tabs to indent, but line 1353 starts with spaces.
Your document appears to use tabs to indent, but line 1354 starts with spaces.
Your document appears to use tabs to indent, but line 1355 starts with spaces.
Your document appears to use tabs to indent, but line 1357 starts with spaces.
Your document appears to use tabs to indent, but line 1358 starts with spaces.
Your document appears to use tabs to indent, but line 1359 starts with spaces.
Your document appears to use tabs to indent, but line 1360 starts with spaces.
Your document appears to use tabs to indent, but line 1361 starts with spaces.
Your document appears to use tabs to indent, but line 1362 starts with spaces.
Your document appears to use tabs to indent, but line 1363 starts with spaces.
Your document appears to use tabs to indent, but line 1364 starts with spaces.
Your document appears to use tabs to indent, but line 1365 starts with spaces.
Your document appears to use tabs to indent, but line 1366 starts with spaces.
Your document appears to use tabs to indent, but line 1367 starts with spaces.
Your document appears to use tabs to indent, but line 1368 starts with spaces.
Your document appears to use tabs to indent, but line 1369 starts with spaces.
Your document appears to use tabs to indent, but line 1370 starts with spaces.
Your document appears to use tabs to indent, but line 1406 starts with spaces.
Your document appears to use tabs to indent, but line 1407 starts with spaces.
Your document appears to use tabs to indent, but line 1419 starts with spaces.
Your document appears to use tabs to indent, but line 1420 starts with spaces.
Your document appears to use tabs to indent, but line 1421 starts with spaces.
Your document appears to use tabs to indent, but line 1429 starts with spaces.
Your document appears to use tabs to indent, but line 1430 starts with spaces.
Your document appears to use tabs to indent, but line 1431 starts with spaces.
Your document appears to use tabs to indent, but line 1432 starts with spaces.
Your document appears to use tabs to indent, but line 1433 starts with spaces.
Your document appears to use tabs to indent, but line 1434 starts with spaces.
Your document appears to use tabs to indent, but line 1435 starts with spaces.
Your document appears to use tabs to indent, but line 1436 starts with spaces.
Your document appears to use tabs to indent, but line 1437 starts with spaces.
Your document appears to use tabs to indent, but line 1438 starts with spaces.
Your document appears to use tabs to indent, but line 1439 starts with spaces.
Your document appears to use tabs to indent, but line 1440 starts with spaces.
Your document appears to use tabs to indent, but line 1441 starts with spaces.
Your document appears to use tabs to indent, but line 1461 starts with spaces.
Your document appears to use tabs to indent, but line 1462 starts with spaces.
Your document appears to use tabs to indent, but line 1493 starts with spaces.
Your document appears to use tabs to indent, but line 1494 starts with spaces.
Your document appears to use tabs to indent, but line 1495 starts with spaces.
Your document appears to use tabs to indent, but line 1496 starts with spaces.
Your document appears to use tabs to indent, but line 1497 starts with spaces.
Your document appears to use tabs to indent, but line 1498 starts with spaces.
Your document appears to use tabs to indent, but line 1499 starts with spaces.
Your document appears to use tabs to indent, but line 1500 starts with spaces.
Your document appears to use tabs to indent, but line 1501 starts with spaces.
Your document appears to use tabs to indent, but line 1513 starts with spaces.
Your document appears to use tabs to indent, but line 1514 starts with spaces.
Your document appears to use tabs to indent, but line 1531 starts with spaces.
Your document appears to use tabs to indent, but line 1532 starts with spaces.
Your document appears to use tabs to indent, but line 1533 starts with spaces.
Your document appears to use tabs to indent, but line 1534 starts with spaces.
Your document appears to use tabs to indent, but line 1627 starts with spaces.
Your document appears to use tabs to indent, but line 1628 starts with spaces.
Your document appears to use tabs to indent, but line 1629 starts with spaces.
Your document appears to use tabs to indent, but line 1820 starts with spaces.
Your document appears to use tabs to indent, but line 1842 starts with spaces.
Your document appears to use tabs to indent, but line 1904 starts with spaces.
Your document appears to use tabs to indent, but line 1905 starts with spaces.
Your document appears to use tabs to indent, but line 2022 starts with spaces.
Your document appears to use tabs to indent, but line 2023 starts with spaces.
Your document appears to use tabs to indent, but line 2024 starts with spaces.
Your document appears to use tabs to indent, but line 2025 starts with spaces.
Your document appears to use tabs to indent, but line 2026 starts with spaces.
Your document appears to use tabs to indent, but line 2027 starts with spaces.
Your document appears to use tabs to indent, but line 2028 starts with spaces.
Your document appears to use tabs to indent, but line 2029 starts with spaces.
Your document appears to use tabs to indent, but line 2030 starts with spaces.
Your document appears to use tabs to indent, but line 2031 starts with spaces.
Your document appears to use tabs to indent, but line 2032 starts with spaces.
Your document appears to use tabs to indent, but line 2033 starts with spaces.
Your document appears to use tabs to indent, but line 2109 starts with spaces.
Your document appears to use tabs to indent, but line 2112 starts with spaces.
Your document appears to use tabs to indent, but line 2115 starts with spaces.
Your document appears to use tabs to indent, but line 2118 starts with spaces.
Your document appears to use tabs to indent, but line 2121 starts with spaces.
Your document appears to use tabs to indent, but line 2480 starts with spaces.
Your document appears to use tabs to indent, but line 2549 starts with spaces.
Your document appears to use tabs to indent, but line 2550 starts with spaces.
Your document appears to use tabs to indent, but line 2551 starts with spaces.
Your document appears to use tabs to indent, but line 2553 starts with spaces.
Your document appears to use tabs to indent, but line 2554 starts with spaces.
Your document appears to use tabs to indent, but line 2555 starts with spaces.
Your document appears to use tabs to indent, but line 2558 starts with spaces.
Your document appears to use tabs to indent, but line 2560 starts with spaces.
Your document appears to use tabs to indent, but line 2565 starts with spaces.
Your document appears to use tabs to indent, but line 2568 starts with spaces.
Your document appears to use tabs to indent, but line 2571 starts with spaces.
Your document appears to use tabs to indent, but line 2574 starts with spaces.
Your document appears to use tabs to indent, but line 2577 starts with spaces.
Your document appears to use tabs to indent, but line 2580 starts with spaces.
Your document appears to use tabs to indent, but line 2583 starts with spaces.
Your document appears to use tabs to indent, but line 2598 starts with spaces.
Your document appears to use tabs to indent, but line 2602 starts with spaces.
Your document appears to use tabs to indent, but line 2607 starts with spaces.
Your document appears to use tabs to indent, but line 2609 starts with spaces.
Your document appears to use tabs to indent, but line 2790 starts with spaces.
WARNING: The heading 'Annex' needs a manually-specified ID.
WARNING: At least one &lt;img> doesn't have its size set (&lt;img bs-line-number="310" src="images/Hypothetical%20IAMF%20Architecture.png" style="width:100%; height:auto;">), but given the type of input document, Bikeshed can't figure out what the size should be.
Either set 'width'/'height' manually, or opt out of auto-detection by setting the 'no-autosize' attribute.
WARNING: Multiple elements have the same ID 'descriptors'.
Deduping, but this ID may not be stable across revisions.
LINK ERROR: No 'property' refs found for 'iamf'.
'iamf'
LINK ERROR: No 'property' refs found for 'base'.
'Base'
LINK ERROR: No 'property' refs found for 'hypotheticalprofile'.
'HypotheticalProfile'
LINK ERROR: No 'property' refs found for 'lpcm'.
'lpcm'
 ✔  Successfully generated, with 4 linking errors
FATAL ERROR: Something prevented me from saving the output document to /tmp/bikeshed_dNms5U:
[Errno 28] No space left on device
 ✘  Did not generate, due to fatal errors
```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20AOMediaCodec/iamf%23625.)._
</details>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1636,7 +1636,7 @@ class decoder_config(ipcm) {
 <dfn noexport>sample_rate</dfn> indicates the sample rate of the input audio in Hz. It SHALL take a value from the set 44.1k, 16k, 32k, 48k, and 96k.
 
 [=audio_frame()=] of Audio_Frame_OBU is only one single PCM audio frame of mono or stereo channels.
-	- In case of that [=audio_frame()=] contains one single PCM audio frame of stereo channels, the ith audio sample of the left channel is followed by the ith audio sample of the right channel, and then the (i+1)th audio sample of the left channel is followed by the (i+1)th audio sample of the right channel, where i = 1, 2, ..., [=num_samples_per_frame=].
+	- In case of that [=audio_frame()=] contains one single PCM audio frame of stereo channels, the ith audio sample of the left channel is followed by the ith audio sample of the right channel, and then the (i+1)th audio sample of the left channel is followed by the (i+1)th audio sample of the right channel, where i = 1, 2, ..., [=num_samples_per_frame=] - 1.
```
Comment (felicialim) [inline index.bs:1639]: why -1?

**Review by felicialim — APPROVED** [review]

File: index.bs [hunk 2]
```diff
@@ -486,11 +486,11 @@ NOTE: This means that if one of the values is set to the number of samples (i.e.
...
 
-This flag SHALL be set to 0 for this version of the specification. An OBU parser that is conformant with this version of the specification SHALL ignore the [=extension_header_bytes=].
+This flag SHALL be set to 0 for this version of the specification. An OBU parser that is conformant with this version of the specification SHOULD ignore the [=extension_header_bytes=].
 
 NOTE: A future version of the specification may use this flag to specify an extension header field by setting [=obu_extension_flag=] = 1 and setting the size of the extended header to [=extension_header_size=].
 
-<dfn noexport>obu_size</dfn> indicates the size in bytes of the OBU immediately following the obu_size field of the OBU. An OBU MAY have extra bytes after consuming all the bytes per the OBU syntax definition, parsers compliant with this version of the specification SHALL ignore the extra bytes. 
+<dfn noexport>obu_size</dfn> indicates the size in bytes of the OBU immediately following the obu_size field of the OBU. An OBU MAY have extra bytes after consuming all the bytes per the OBU syntax definition, parsers compliant with this version of the specification SHOULD ignore the extra bytes.
```
Comment (sunghee-hwang) [inline index.bs:493]:
```suggestion
<dfn noexport>obu_size</dfn> indicates the size in bytes of the OBU immediately following the obu_size field of the OBU. An OBU MAY have extra bytes after consuming all the bytes per the OBU syntax definition. Parsers compliant with this version of the specification SHOULD ignore the extra bytes.
```

File: index.bs [hunk 3]
```diff
@@ -477,20 +477,20 @@ For a given coded [=Audio Substream=],
...
 
-NOTE: This means that if one of the values is set to the number of samples (i.e. [=num_samples_per_frame=]) in the [=Audio Frame OBU=], the other value is set to 0.
+NOTE: This means that if one of the values is set to the number of samples in the [=Audio Frame OBU=] (i.e., [=num_samples_per_frame=]), the other value is set to 0.
 
-- When [=num_samples_to_trim_at_start=] is non-zero, all [=Audio Frame OBU=]s with the same [=audio_substream_id=], and preceding this OBU back until the [=Codec Config OBU=] defining this [=Audio Substream=], SHALL have their [=num_samples_to_trim_at_start=] field equal to the number of samples (i.e. [=num_samples_per_frame=]) in the corresponding Audio Frame OBU.
+- When [=num_samples_to_trim_at_start=] is non-zero, all [=Audio Frame OBU=]s with the same [=audio_substream_id=], and preceding this OBU back until the [=Codec Config OBU=] defining this [=Audio Substream=], SHALL have their [=num_samples_to_trim_at_start=] field equal to the number of samples in the corresponding Audio Frame OBU (i.e., [=num_samples_per_frame=]).
```
Comment (sunghee-hwang) [inline index.bs:484]:
```suggestion
- When [=num_samples_to_trim_at_start=] is non-zero, all [=Audio Frame OBU=]s with the same [=audio_substream_id=], and preceding this OBU back until the [=Codec Config OBU=] defining this [=Audio Substream=], SHALL have their [=num_samples_to_trim_at_start=] field equal to the number of samples in the corresponding [=Audio Frame OBU=] (i.e., [=num_samples_per_frame=]).
```

File: index.bs [hunk 4]
```diff
@@ -477,20 +477,20 @@ For a given coded [=Audio Substream=],
...
 
-This flag SHALL be set to 0 for this version of the specification. An OBU parser that is conformant with this version of the specification SHALL ignore the [=extension_header_bytes=].
+This flag SHALL be set to 0 for this version of the specification. An OBU parser that is conformant with this version of the specification SHOULD ignore the [=extension_header_bytes=].
 
 NOTE: A future version of the specification may use this flag to specify an extension header field by setting [=obu_extension_flag=] = 1 and setting the size of the extended header to [=extension_header_size=].
 
-<dfn noexport>obu_size</dfn> indicates the size in bytes of the OBU immediately following the obu_size field of the OBU. An OBU MAY have extra bytes after consuming all the bytes per the OBU syntax definition, parsers compliant to this version of the specification SHALL ignore the extra bytes. 
+<dfn noexport>obu_size</dfn> indicates the size in bytes of the OBU immediately following the obu_size field of the OBU. An OBU MAY have extra bytes after consuming all the bytes per the OBU syntax definition. Parsers compliant with this version of the specification SHOULD ignore the extra bytes.
```
Comment (sunghee-hwang) [inline index.bs:493]:
```suggestion
<dfn noexport>obu_size</dfn> indicates the size in bytes of the OBU immediately following the obu_size field of the OBU. An OBU MAY have extra bytes after consuming all the bytes per the OBU syntax definition. Parsers compliant with this version of the specification SHOULD ignore the extra bytes. 
```

**Review by sunghee-hwang — COMMENTED** [review]
To fix pre-view error

**Pull request merged**